doc/classes/FileDialog.xml

```diff
@@ -56,6 +58,7 @@
 	</methods>
 	<members>
 		<member name="access" type="int" setter="set_access" getter="get_access" enum="FileDialog.Access">
+			See enum [code]Access[/code] constants.
```

Comment:
I would recommend trying to explain what this does, just to get a sense for what "access" means. Especially as the constants are not documented very well at the moment.

Reply:
I will try to explain it a bit. The constants looked like a good description to me.